### DISCUSSION TITLE
Unchecked option but database keeps locking itself

### DISCUSSION THREAD

**Discussion by Pyanne** [keepassxreboot/keepassxc#9090] [Q&A] [answered]
I use KeepassXC on MacOS 12.6.3. I would like to decide myself when to lock or unlock the database. Although the option is unchecked, it locks itself after a while.

Is this a bug or am I the one who didn't understand how to do it?

![Screenshot 2023-02-08 at 21 39 58](https://user-images.githubusercontent.com/11357449/217650717-fe9700fb-a93c-4ad8-9265-69a9bcce4a24.png)

**Comment by droidmonkey** [comment, marked as answer]
Uncheck the lock when screen turns off or lid is closed

**Comment by vincent-legoll** [comment]
Maybe the option text should be changed, if it's not only "when session is locked or lid is closed" to better reflect what it really is doing

**Reply by droidmonkey** [reply]
I'm just offering another reason why his database would lock automatically

**Reply by vincent-legoll** [reply]
Yeah, I assumed the OP did not close the lid nor lock his session but that the database lock happened anyways. Sorry
